pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -171,6 +171,9 @@ def fractional_octave_bands(
         order=14):
     """Create and/or apply an energy preserving fractional octave filter bank.
 
+    The filters are in accordance with IEC 61260-1:2014 [#]_ for octave and
+    third band widths and the default filter order of 14 (see examples below).
+
```

Comment:
I'm actually not sure if our implementation with hp filters if the band spans across Nyquist is compliant with the standard. Could you please check/confirm if it complies. Otherwise we'd need to add a comment/note here.